tests/app/questionnaire/test_value_source_resolver.py

```diff
@@ -35,6 +35,7 @@ def get_value_source_resolver(
     answer_store: AnswerStore = AnswerStore(),
```

Comment:
This file has tests for each of the value source, so a new one should be added for `response_metadata`.